docs/aerodromes/Amberley.md

```diff
@@ -4,7 +4,78 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## YAMB Positions
 
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Amberley ADC    | Amberley Tower  | 118.300         | AM_TWR        |
+| Amberley SMC    | Amberley Ground  | 129.350      | AM_GND        |
+| Amberley ACD    | Amberley Delivery  | 134.600         | AM_DEL       |
+| Amberley ATIS    |   | 123.300         | YAMB_ATIS       |
+
+
+
+## Airspace
+
+AM ADC owns the Class C airspace in the AM CTR within 10nm of the YAMB ARP from `SFC` to `A015`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+
+## YAMB Aerodrome
+
+### SID Selection
+Aircraft planned via **BN**, **JEDDA**, **MESED**, **BOBOP**, **TATEN**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR sid or a visual departure. 
+
+#### Coded Departures
+Visual Departures are commonly requested which are often in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for Amberley. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane avoiding civilian traffic inbound to Brisbane. 
+
+Aircraft will make a visual departure in the circuit direction and fly overhead Amberley to begin tracking for their initial waypoint.
+
+!!! example
+    WOLF03 was assiged the BYRON7 departure with their intial clearance. 
+    **AM ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
+    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
+
+| Departure | Waypoint | Constraint |
+| --------- | ---------| ----------|
+| Northern1 | BINUP  | `F190` |
+| Central1 | BINUP | `F190` |
+| Byron7 | COWIE | `F190` |
```

Comment:
For standardisation, could we put a space between the departure name and the number (e.g. Northern 1)? Also I assume from the example the constraint refers to the waypoint in the column to the left? Maybe it could be a single 'Constraint' column with '`F190` or above by BINUP', etc

Not sure if it's a lot of work, but if we could replicate those routes here, that would make it easier for controllers to reference.